### ISSUE TITLE
Typing Error

### ISSUE THREAD

**Issue by Kayxue** [AngeloCore/discord-buttons#79]
Version of discord-buttons: 3.2.1

![image](https://user-images.githubusercontent.com/26020165/121193077-1c4c0a00-c8a0-11eb-8538-ec562be1307d.png)

**Comment by AngeloCore** [comment]
Whats the error?

**Comment by Kayxue** [comment]
![image](https://user-images.githubusercontent.com/26020165/121193858-c330a600-c8a0-11eb-95af-57304c1a48ec.png)
![image](https://user-images.githubusercontent.com/26020165/121193916-d17ec200-c8a0-11eb-9f9f-dfc247509ecc.png)
![image](https://user-images.githubusercontent.com/26020165/121193989-e52a2880-c8a0-11eb-8b69-4eda0dd3247e.png)

**Comment by Kayxue** [comment]
1.Array[MessageActionRaw] => Array<MessageActionRaw>

2.CollectorFIlter: Idk how to fix, the error message said "CollectorFilter is not genetic"

3.Array[MessageButton] => Array<MessageButton>

4.ExtendedTextChannel, ExtendedDMChannel and ExtendedNewsChannel: Idk how to fix, Error messages:

```
Interface 'ExtendedTextChannel' incorrectly extends interface 'TextChannel'.
  Types of property 'send' are incompatible.
    Type '(content: MessageAdditions | APIMessageContentResolvable | (ExtendedMessageOptions & { split?: false; })) => Promise<...>' is not assignable to type '{ (content: MessageAdditions | APIMessageContentResolvable | (MessageOptions & { split?: false; })): Promise<Message>; (options: MessageOptions & { ...; }): Promise<...>; (options: MessageOptions | APIMessage): Promise<...>; (content: any, options: MessageAdditions | (MessageOptions & { ...; })): Promise<...>; (cont...'.
      Types of parameters 'content' and 'options' are incompatible.
        Type 'MessageOptions & { split: true | SplitOptions; }' is not assignable to type 'MessageAdditions | APIMessageContentResolvable | (ExtendedMessageOptions & { split?: false; })'.
          Type 'MessageOptions & { split: true | SplitOptions; }' is not assignable to type 'ExtendedMessageOptions & { split?: false; }'.
            Type 'MessageOptions & { split: true | SplitOptions; }' is not assignable to type '{ split?: false; }'.
              Types of property 'split' are incompatible.
                Type '(boolean | SplitOptions) & (true | SplitOptions)' is not assignable to type 'false'.
                  Type 'true' is not assignable to type 'false'.ts(2430)
```

**Comment by Unluckymichell** [comment]
I have corrected the typings in this PR (#83)! Should solve the above problem

**Comment by AngeloCore** [comment]
Check #80
